src/transform/CollapseTable.css

```diff
@@ -89,14 +89,14 @@ table.infobox caption {
   clear: both;
 }
 
-.pagelib-theme-dark .app_table_container {
-  box-shadow: none;
-  background-color: #202020;
+.pagelib-theme-dark div.app_table_container {
+  box-shadow: none !important;
```

Comment:
Kind of the same question as above. I thought this was the pagelib's class. Who are we overriding with `!important`?